glotaran/plugin_system/data_io_registration.py

```diff
@@ -249,20 +286,35 @@ def show_data_io_method_help(
     show_method_help(io, method_name)
 
 
-def data_io_plugin_table() -> str:
+def data_io_plugin_table(*, plugin_names: bool = False, full_names: bool = False) -> str:
     """Return registered data io plugins and which functions they support as markdown table.
 
     This is especially useful when you work with new plugins.
 
+    Parameters
+    ----------
+    plugin_names:bool
+        Whether or not to add the names of the plugins to the table.
+    full_names: bool
```

Comment:
fyi, you mix and match
a: b
and
a : b

Reply:
Yeah saw that now that you pointed it out 😅 
`a : b` comes from the autogenerated docstring by from the VSCode extension [Python Docstring Generator
](https://github.com/NilsJPWerner/autoDocstring)
`a: b` Is from copy-pasting out of the signature
`a:b` Is from copy-pasting out of the signature, before saving the file an black properly formatting it

I want an autformatter for docstrings! 😆